demisto_sdk/commands/common/hook_validations/id.py

```diff
@@ -69,7 +67,7 @@ def load_id_set(self):
 
             return id_set
 
-    def is_valid_in_id_set(self, file_path: str, obj_data: OrderedDict, obj_set: list):
+    def _is_valid_in_id_set(self, file_path: str, obj_data: OrderedDict, obj_set: list):
```

Comment:
why is this change required?

Reply:
not required, but better to indicate that this func is private.